### PR TITLE
Clarify that custom events do not bubble

### PR DESCRIPTION
**Pull request by amcsi**
This may be a bit wordy, but I remember when I was first working with VueJS, it confused me that custom events didn't bubble up. I thought I could listen to custom events on any grandparent element.

With these changes, I would like to have this pointed out to people if I'm not alone on this.

The text I used might be a bit wordy, so feel free to suggest changes.

### CODE REVIEW

**Comment by amcsi** [comment]
Would someone please provide some feedback about what they think about this PR?

**Comment by sdras** [comment]
Thanks @amcsi! Sorry about the delay.

I think this a great addition! But where it is right now breaks up the flow of the tutorial, it really belongs in the caveats section at the bottom of the page. Here are the things we would need to accept the PR:

- Moved to the caveats section
- Take it out of a tip
- It would be good to mention `$listeners` as an alternative (let me know if you need an example, or I can follow it up with another PR to clarify)

If this all seems like too much to do after all this time, totally understandable, just ping me here and I'll close it out and open a new one.

**Comment by amcsi** [comment]
@sdras thanks for the reply! That's fair enough. I don't think I will have time to do it any time soon, so feel free to close this PR for now.
